liam2/simulation.py

```diff
@@ -100,6 +100,7 @@ class Simulation(object):
     yaml_layout = {
         'import': None,
         'globals': {
+            'DEBUG': None,

```

Comment:
please revert this line. A debug flag usable in user code would make sense, but having it in globals seems weird. Globals (in my opinion) are for user/model-defined stuff, while debug should be set by the system (e.g. the command-line argument).